bridge/engine/ActiveMediaList.cpp

```diff
@@ -355,11 +387,25 @@ void ActiveMediaList::process(const uint64_t timestampMs, bool& outDominantSpeak
 
     auto dominantSpeaker = heap.top();
 
+    assert(_c9_conference || 0 == activeTalkersSnapshot.count);
+
     for (size_t i = 0; i < _audioLastN && !heap.empty(); ++i)
     {
-        updateActiveAudioList(heap.top().participant);
+        const auto& top = heap.top();
+        updateActiveAudioList(top.participant);
+        if (!_c9_conference)
+        {
```

Comment:
Perhaps && instead of nesting

Reply:
Isn't it a bit strange that I understood what you wrote actually :) Fixed.